website/docs/d/cloud_project_user.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "ovh"
+page_title: "OVH: ovh_cloud_project_user"
+sidebar_current: "docs-ovh-datasource-cloud-project-user"
+description: |-
+  Get the details of a public cloud project user.
+---
+
+# ovh_cloud_project_user
+
+Get the user details of a previously created public cloud project user.
+
+## Example Usage
+
+```hcl
+resource "ovh_cloud_project_user" "user" {
```

Comment:
You don't have to create one previously on terraform in order to use the datasource.
The goal of a datasource is originally to retrieve an existing resource in the platform :).
So this doc is for datasource so only focus to the datasource, not the resource :).